llvm/lib/Analysis/DXILResource.cpp

```diff
@@ -879,8 +883,122 @@ SmallVector<dxil::ResourceInfo *> DXILResourceMap::findByUse(const Value *Key) {
 
 //===----------------------------------------------------------------------===//
 
+void DXILResourceBindingInfo::populate(Module &M, DXILResourceTypeMap &DRTM) {
+  struct Binding {
+    ResourceClass ResClass;
```

Comment:
I'd probably just call this "RC", we use that abbreviation pretty commonly elsewhere